src/core/HazelcastError.ts

```diff
@@ -75,6 +77,32 @@ export class HazelcastSerializationError extends HazelcastError {
     }
 }
 
+export class SchemaNotFoundError extends HazelcastError {
+
+    schemaId: Long;
+
+    constructor(msg: string, schemaId: Long, cause?: Error, serverStackTrace?: ServerErrorStackElement[]) {
```

Comment:
`SchemaNotFoundError` is public isn't it? We can throw it to the user but not `SchemaNotReplicatedError`.  About the readonly thing, I think I wanted to be sure that it is only set once because this is how it is intended to be used in controlled serialization. I think it is the same in `SchemaNotFoundError` as well. Let me change it to readonly as well.